### PR TITLE
[aotprof-tool] Initial import

### PR DESCRIPTION
**Pull request by radekdoulik**
Imported https://github.com/radekdoulik/aotprofile-tool and updated it
to use Mono.Profiler.Log assembly

### CODE REVIEW

**Comment by radekdoulik** [comment]
@monojenkins build failed

File: man/aotprof-tool.1 [hunk 1]
```diff
@@ -0,0 +1,91 @@
+.TH aotprof-tool 1
```
Comment (vargaz) [inline man/aotprof-tool.1]: Should a tool's name include the -tool suffix ?

**Comment by lewing** [comment]
@monojenkins build failed

File: man/aotprof-tool.1 (hunk 1 above)
Comment (marek-safar) [reply man/aotprof-tool.1]: It should not, @radekdoulik could come up with a better name?

**Comment by radekdoulik** [comment]
Bunch of the tools in mcs/tools have util suffix, so *aotprofutil* or *aprofutil* might work.

Other possible names:
*aprofcruncher*
*aprofblender*
*aprofninja*
*aprofinspector*

**Comment by steveisok** [comment]
I'm awful w/ names.  I like aprofinspector

**Comment by marek-safar** [comment]
I like the aprof prefix, perhaps simple `aprofutil` would work

**Comment by radekdoulik** [comment]
OK, let use the short version then.

**Comment by steveisok** [comment]
@monojenkins build failed

**Comment by akoeplinger** [comment]
@monojenkins commit csproj

**Pull request merged**